stl/inc/xstring

```diff
@@ -2588,19 +2606,39 @@ public:
     }
 
 private:
-    static constexpr void _Start_element_lifetimes(_Elem* const _Ptr, const size_type _Size) {
+    enum class _Reallocation_policy { _At_least, _Exactly };

```

Comment:
Thanks for your help! As to the new changes, I think `_Allocation_policy` will be a better name here, as `_Allocate_for_capacity` only involves allocation logic. (vector uses `_Reallocation_policy` for `_Reallocate`)

Reply:
You're welcome! And great catch, I wasn't thinking enough as I was copy-pasting that code. I've pushed a rename.